ts/components/wallet/card/index.tsx

```diff
@@ -64,7 +75,15 @@ const styles = StyleSheet.create({
   }
 });
 
-export type Props = Readonly<{
+type ReduxMappedStateProps = Readonly<{
+  isFavoriteCard: boolean;
+}>;
+
+type ReduxMappedDispatchProps = Readonly<{
+  setFavoriteCard: (item: Option<number>) => void;
```

Comment:
just a suggestion, it would be awesome to use a tagged type for the card ID, instead of just `number` (see an example of a tagged type [here](https://github.com/teamdigitale/italia-app/blob/master/ts/types/SessionToken.ts))